spec/furniture/marketplace/orders_controller_request_spec.rb

```diff
@@ -24,8 +24,9 @@
     end
 
     context "when the order is not viewable by the current person" do
+      before { sign_in(space, create(:person)) }
```

Comment:
For my learning, this `create(:person)` call is just creating a guest not associated with a space (as _member_, per se), right?

Reply:
Kinda! This is creating a `Person` record in the database; which is not a `Guest` (`Guests` are unpersisted). You can see the factory definition in [spec/factories/people.rb](https://github.com/zinc-collective/convene/blob/main/spec/factories/person.rb)

If we want to create a `Person` who is `Member` of a `Space`, we would use the [`create(:membership)` factory](https://github.com/zinc-collective/convene/blob/main/spec/factories/membership.rb)

Reply:
Oh oh I get it. You aren't trying to simulate a "guest" as I thought, because you're persisting a record. This is testing that a person cannot view another person's order. 👍🏻